contracts/wallet.sol

```diff
@@ -586,4 +596,136 @@ contract Wallet is Vault {
             _topUpAvailable = topUpLimit;
         }
     }
+
+    /// @dev This function allows for the owner to send transaction from the Wallet to arbitrary addresses
+    /// @param _destination address of the transaction
+    /// @param _value ETH amount in wei
+    /// @param _data transaction payload binary
+    function executeTransaction(address _destination, uint _value, bytes _data) external onlyOwner {
+
+        require(address(_destination).isContract(), "executeTransaction: call to non-contract");
```

Comment:
we should not do this - we should be able to send at least ETH to other addresses

Reply:
if we want to send ETH we have the transfer function, why would somebody use this one. Additionally, it would contradict the "require(msg.data.length == 0, "data in fallback");" we have in our fallback, expecting that when a user sends both value and data something might be wrong.

Reply:
This has been resolved. Users do have to be explicit about whether or not they are interacting with a contract or an externally owned address when they call executeTransaction. This way we shoudl minimise the potential issues.